kubespawner/spawner.py

```diff
@@ -433,6 +442,15 @@ def _hub_connect_port_default(self):
         """
     )
 
+    user_storage_match_labels = Dict(
+        help="""
```

Comment:
Ok, its gone now. Yes, it was for that case where you already had existing PVs. This is the way I have always used PV + PVC, more manual than the automatic creation.